src/react/workflow/replication/Replication.jsx

```diff
@@ -118,7 +125,7 @@ function Replication({ streams, bucketList, locations, workflowDetails, showEdit
                     </T.GroupName>
                     <T.GroupContent>
                         <T.Row>
-                            <T.Key> Name </T.Key>
+                            <T.Key required={createMode}> Rule Name </T.Key>
```

Comment:
Name is not required, we generate one for you if missing. CC @Cuervino

Reply:
Ah, well spotted.

Reply:
done